dev-config.yaml

```diff
@@ -0,0 +1,11 @@
+# FIXME: When running tests, we will need a local container registry to test
+#        pushing images to that can be reached from the build pods.
+#
```

Comment:
@consideRatio, is this the only bit that's currently missing from this PR?

Reply:
Its missing, but it doesn't have to be in this PR I figure!

I'll dig into this PR once again now this afternoon. I just finished with a new hub deployment and will eat some food.

Reply:
No hurry! Thank you! let me know if there's anything I can help with

Reply:
We now test and push against a local registry!